res/database/Default/database_ads.xml

```diff
@@ -1368,11 +1370,11 @@ Brucey - harsh fragrance for male experts.</en>
 		<ad id="ronny-ad-litfasssaeule-01" creator="5578" created_by="Ronny">
 			<title>
 				<de>Litfaßsäule</de>
-				<en>Morris Column</en>
+				<en>Adpillar</en>
```

Comment:
because "Morris Column" is not all too widely used (wikipedia redirects, no mention of that term e.g. on https://www.google.com/doodles/celebrating-the-litfasaule ), "Adpillar" is self-explanatory, and the original German also uses the most common and everyday word for that thing.

Reply:
OK